### ISSUE TITLE
Meta keywords are stripped while uploading images on magento 2.3.1

### ISSUE THREAD

**Issue by utka955** [magento/magento2#25186]
### Preconditions (*)
1. Magento 2.3.x & 2.3-develop
2. PHP 7.x
3. Have an image with keywords 
![image](https://user-images.githubusercontent.com/51680850/67370036-b43d9880-f582-11e9-931d-512006b90cd4.png)

### Summary (*)
I am having SEO optimized images that contain keywords and when I upload them on Magento and download the image then all the keywords are missing. Right now it is using GD2 but I tried with imagemagick also.

### Steps to reproduce (*)
1. Go to the create product page
2. In the Content tab upload test image with keywords to the Description(in wysiwyg editor)
Image e.g.:
![france](https://user-images.githubusercontent.com/51680850/67369923-85272700-f582-11e9-9192-9ccd6f1c07a8.jpg)
![image](https://user-images.githubusercontent.com/51680850/67370338-24e4b500-f583-11e9-9b03-7ad0a4e764dc.png)
3. Right click on just uploaded image and save it 
4. Check for keywords 

### Expected result (*)
All the images should have the keywords after upload also.

### Actual result (*)
Image keywords are stripped.
![image](https://user-images.githubusercontent.com/51680850/67370578-96246800-f583-11e9-8f24-a38b1142a86e.png)

**Comment by magento-engcom-team** [comment]
:white_check_mark: Confirmed by @engcom-Charlie 
 Thank you for verifying the issue. Based on the provided information internal tickets `MC-22029` were created

**Issue Available**: @engcom-Charlie, _You will be automatically unassigned. Contributors/Maintainers can claim this issue to continue. To reclaim and continue work, reassign the ticket to yourself._

**Comment by devamitbera** [comment]
The issue is produced at Magento at 2.4-develop' branch.

**Original:**


 
![Screenshot from 2020-02-02 13-51-38](https://user-images.githubusercontent.com/6008488/73605347-292d5f00-45c3-11ea-8ce9-c34293b79254.png)


**after uploaded**

![Screenshot from 2020-02-02 13-50-56](https://user-images.githubusercontent.com/6008488/73605337-1155db00-45c3-11ea-8533-a81850d32d15.png)

**Comment by engcom-Kilo** [comment]
@devamitbera @utka955 
Checked with fresh ver. dev-2.4-develop and Image saved properly. Meta-data same as in original uploaded Image.

**Comment by lurajcevi** [comment]
@magento give me 2.4-develop instance

**Comment by magento-engcom-team** [comment]
Hi @lurajcevi. Thank you for your request. I'm working on Magento 2.4-develop instance for you

**Comment by magento-engcom-team** [comment]
Hi @lurajcevi, here is your [Magento instance](https://i-25186-2-4-develop.instances.magento-community.engineering).
Admin access: [https://i-25186-2-4-develop.instances.magento-community.engineering/admin_dd81](https://i-25186-2-4-develop.instances.magento-community.engineering/admin_dd81)
Login: `584c9621` Password: `39acca8e0a9b`
Instance will be terminated in up to 3 hours.

**Comment by lurajcevi** [comment]
@devamitbera @utka955 @engcom-Kilo 
Also checked with fresh 2.4 develop version, and was unable to reproduce this.
Uploaded image (and downloaded afterwards) keeps its 'keywords' metadata.

This was checked with `exiftool` on linux

```
└─[$] <> exiftool initial_uploaded.jpg| grep Key
Keywords                        : france
```

**Comment by sdzhepa** [comment]
Closed as already fixed in `2.4-develop`
